### ISSUE TITLE
Failed to attach (similar to #23)

### ISSUE THREAD

**Issue by eris-webserv** [fritzlb/iOS17-JIT-WIN#87]
I'm atempting to enable JIT on DolphiniOS. I signed and installed the app with Feather without modifying the bundle ID and used the bundle ID feather displayed. The app reopened, but JIT was not enabled due to the same error as #23.

**I am using a paid developer certificate from Kravasign, not an enterprise certificate.**

**Comment by JojoZockt** [comment]
wrong type of paid developer certificate. you need a certificate with the get-task-allow entitlement which is a development certificate. kravasign only supports distribution certificates right now. you need to use other signing services or buy a paid developer account from apple.

**Comment by eris-webserv** [comment]
Yeah I figured THAT out a while ago. They should really clear that up.
